org-wide-skills/contributor-onboarding/SKILL.md

```
@@ -0,0 +1,55 @@
+---
+name: aossie-contributor-onboarding
+description: Org-level entry gate for all AOSSIE contributors. Use when a contributor asks "how do I start", "what should I read", or begins any contribution-related conversation.
+---
+
+A skill serving as the org-wide entry gate for all contributors to AOSSIE repositories.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)